codegen/core/src/main/scala/com/lightbend/akkasls/codegen/DescriptorSet.scala

```diff
@@ -49,10 +49,8 @@ object DescriptorSet {
   @SuppressWarnings(Array("org.wartremover.warts.Throw"))
   def fileDescriptors(
       file: File
-  ): Either[CannotOpen, Iterable[Either[ReadFailure, Descriptors.FileDescriptor]]] =
-    Using[FileInputStream, Either[CannotOpen, Iterable[
-      Either[ReadFailure, Descriptors.FileDescriptor]
-    ]]](
+  ): Either[CannotOpen, Either[ReadFailure, Iterable[Descriptors.FileDescriptor]]] =
```

Comment:
So now if we fail to read one file we throw away all results? That might be OK if we're sure we need all of them anyway

Reply:
We bail out on any error now, before it could live with certain descriptors failing.